sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/v2/Spark33XShims.scala

```diff
@@ -56,4 +60,86 @@ trait Spark33XShims extends Spark33XFileOptionsShims {
     new ParquetFilters(schema, pushDownDate, pushDownTimestamp, pushDownDecimal, pushDownStartWith,
       pushDownInFilterThreshold, caseSensitive, datetimeRebaseMode)
   }
+
+  override def getScans: Map[Class[_ <: Scan], ScanRule[_ <: Scan]] = Seq(
+    GpuOverrides.scan[ParquetScan](
+      "Parquet parsing",
+      (a, conf, p, r) => new ScanMeta[ParquetScan](a, conf, p, r) {
+        override def tagSelfForGpu(): Unit = {
+          GpuParquetScanBase.tagSupport(this)
+          // we are being overly cautious and that Parquet does not support this yet
+          if (a.isInstanceOf[SupportsRuntimeFiltering]) {
+            willNotWorkOnGpu("Parquet does not support Runtime filtering (DPP)" +
+              " on datasource V2 yet.")
+          } else if (a.pushedAggregate.nonEmpty) {
+            willNotWorkOnGpu(
+              "Implementing aggregate push down on GPU gains little performance improvement"
```

Comment:
nit: If it gains some then we should do it, but I don't think it would gain any, and we cannot make that statement until we have built and tested it. I would say something more like.

"Aggregates pushed into parquet read, which is a metadata only operation"

Reply:
Capitalization nits: the string passed to `willNotWorkOnGpu` is appended to form the sentence that will be shown to the user, so it should not normally start with a capitalized word.  Also Parquet is a proper noun, so it should be capitalized.

Reply:
Thanks for your reviews, I will update it.